test/unit/3dtileslayerstyle.js

```diff
@@ -7,79 +7,104 @@ import PlanarView from 'Core/Prefab/PlanarView';
 import C3DTBatchTable from 'Core/3DTiles/C3DTBatchTable';
 import C3DTilesSource from 'Source/C3DTilesSource';
 import C3DTilesLayer from 'Layer/C3DTilesLayer';
+import sinon from 'sinon';
+import Fetcher from 'Provider/Fetcher';
 import Renderer from './bootstrap';
 
+import tileset from '../data/3dTiles/tileset.json';
```

Comment:
This file is 40kb and could be removed. Seems we can instantiate a `C3DTilesSource` with an empty tileset in iTowns.

```suggestion
const tileset = {};
```

Reply:
Done